problems/sevenBoom.js

```diff
@@ -11,8 +11,12 @@
 * 17 is also replaced with 'BOOM' because it contains a 7.
 */
 
-function sevenBoom() {
-
+function sevenBoom(n) {
+    let newArr = []
+    for(let i = 1; i <= n; i++){
+        i % 7 === 0 || i % 10 === 7 ? newArr.push("BOOM") : newArr.push(i)
```

Comment:
Consider numbers 70-79. Will this function push "BOOM" for any number that contains a 7?